src/test/python/jointGenerator/rockInput.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+import numpy as np
+import sys
+from Joint import Joint
+import math
+
+# Reads input text file containing data on rock volume and joints
+# Outputs text file that can be read by sparkRocks
+
+# Process command line arguments:
+inputFile = open(str(sys.argv[1]), 'r')
```

Comment:
I know this is mostly an internal utility file, but I think you should check for the proper number of command line arguments and print out an error message when needed.